src/scripts/pbs_status.py

```diff
@@ -241,7 +241,7 @@ def qstat(jobid=""):
         command += ('-1',)  # -1 conflicts with -f in PBS Pro
     if jobid:
         command += ('-x', jobid) if pbs_pro else (jobid,)
-    qstat_proc = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    qstat_proc = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, universal_newlines=True)
```

Comment:
It also means something completely different on Python 2 vs 3. I'm guessing what you want it for is getting a `str` instead of a `bytes` out of the output in Python 3? It will do the job but unless you trust qstat to produce valid unicode, you run into the risk of having a UnicodeDecodeError when you try to read.

Reply:
>  Not your fault of course, but universal_newlines is such a bad name for this option...

Python 3.7 introduces a new option called `text` that does the same thing so they had the same thought :) 

> It will do the job but unless you trust qstat to produce valid unicode, you run into the risk of having a UnicodeDecodeError when you try to read.

Should I decode the bytestring by hand with `latin-1` instead?

Reply:
I dunno about the `text` option; I think the Python 3 way to do it is to pass the `encoding` parameter, but that is not supported in Python 2. Decoding the bytestring by yourself is safer (you'll want to skip doing that on Python 2 of course).

Reply:
Yeah, i remember reading that the new `text` option is just an alias for `universal_newlines`.  I kind of like the `text` name better, given what it actually does; but yeah i'd guess it's probably too new to use it.

\<rant>
I mean, to some extent i get it.  The C standard library concept of "text" is centered around the concept of what constitutes a newline (`\r\n` in dos/windows, `\n` in unix), vs "binary" where newline bytes are uninterpreted.  But in python3, "text" means unicode (so it implies character set decoding) and "binary" means bytes.  But unicode decoding is different from newline interpretation, and the option name `universal_newlines` seems to conflate the two concepts, without really making it clear that it decodes unicode.
\</rant>

Reply:
@matyasselmeci how's this look? https://github.com/htcondor/BLAH/pull/27/commits/b44ee154d6507759bb924dc87e285b5e1af1ee29